libs/cdk/project.json

```diff
@@ -25,6 +25,45 @@
       }
     },
     "build": {
+      "executor": "@nrwl/workspace:run-commands",
+      "dependsOn": [
+        {
+          "target": "build-base",
+          "projects": "self"
+        },
+        {
+          "target": "build-schematics",
+          "projects": "self"
+        }
+      ],
```

Comment:
we could configure this as general taskDependency in `nx.json` what do u think?

Reply:
There's an issue with Nx, if I change `cdk:build-base`  to `template:build-lib`  which is the same as the `template:build-lib` then it breaks the build, we cannot use global dependsOn config for now.